phoenix-tools/src/it/java/org/apache/phoenix/schema/SchemaExtractionToolIT.java

```diff
@@ -58,156 +130,251 @@ public void testCreateTableStatement_tenant() throws Exception {
+        Assert.assertEquals(query.toUpperCase(), result.toUpperCase());
     }
 
     @Test
-    public void testCreateViewStatement() throws Exception {
+    public void testCreateTableWithArrayColumn() throws Exception {
         String tableName = generateUniqueName();
         String schemaName = generateUniqueName();
-        String viewName = generateUniqueName();
-        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
-        String properties = "TTL=2592000,IMMUTABLE_ROWS=true,DISABLE_WAL=true";
+        String pTableFullName = SchemaUtil.getQualifiedTableName(schemaName, tableName);
+        String query = "create table " + pTableFullName +
+                "(a_char CHAR(15) NOT NULL, " +
+                "b_char CHAR(10) NOT NULL, " +
+                "c_var_array VARCHAR ARRAY, " +
+                "d_char_array CHAR(15) ARRAY[3] CONSTRAINT PK PRIMARY KEY (a_char, b_char)) " +
+                "TTL=2592000, IMMUTABLE_STORAGE_SCHEME=ONE_CELL_PER_COLUMN, REPLICATION_SCOPE=1";
+        List<String> queries = new ArrayList<String>(){};
+        queries.add(query);
+        String result = runSchemaExtractionTool(schemaName, tableName, null, queries);
+        Assert.assertEquals(query.toUpperCase(), result.toUpperCase());
+    }
 
-        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+    @Test
+    public void testCreateTableWithNonDefaultColumnFamily() throws Exception {
+        String tableName = generateUniqueName();
+        String schemaName = generateUniqueName();
+        String pTableFullName = SchemaUtil.getQualifiedTableName(schemaName, tableName);
+        String query = "create table " + pTableFullName +
+                "(a_char CHAR(15) NOT NULL, " +
+                "b_char CHAR(10) NOT NULL, " +
+                "\"av\".\"_\" CHAR(1), " +
+                "\"bv\".\"_\" CHAR(1), " +
+                "\"cv\".\"_\" CHAR(1), " +
+                "\"dv\".\"_\" CHAR(1) CONSTRAINT PK PRIMARY KEY (a_char, b_char)) " +
+                "TTL=1209600, IMMUTABLE_ROWS=true, IMMUTABLE_STORAGE_SCHEME=ONE_CELL_PER_COLUMN, SALT_BUCKETS=16, MULTI_TENANT=true";
+        List<String> queries = new ArrayList<String>(){};
+        queries.add(query);
+        String result = runSchemaExtractionTool(schemaName, tableName, null, queries);
+        Assert.assertEquals(query.toUpperCase(), result.toUpperCase());
+    }
 
-            String pTableFullName = SchemaUtil.getQualifiedTableName(schemaName, tableName);
-            conn.createStatement().execute("CREATE TABLE "+pTableFullName + "(k BIGINT NOT NULL PRIMARY KEY, "
-                    + "v1 VARCHAR, v2 VARCHAR)"
-                    + properties);
-            String viewFullName = SchemaUtil.getQualifiedTableName(schemaName, viewName);
-            String viewFullName1 = SchemaUtil.getQualifiedTableName(schemaName, viewName+"1");
+    @Test
+    public void testCreateTableWithUniversalCFProperties() throws Exception {
```

Comment:
I feel this and testCreateTableWithNonDefaultColumnFamily can be clubbed into one test?